emojis.py

```
@@ -0,0 +1,92 @@
+            if ext not in (".png", ".gif"):
+                continue
+            emoji_name = name
+            try:
+                with open(os.path.join(root, filename), "rb") as image:
+                    await client.create_application_emoji(name=emoji_name, image=image.read())
+            except discord.HTTPException as e:
+                print(f"couldn't upload {emoji_name}: {e}")
+
+
+async def main() -> None:
+    subprocess.run(["git", "clone", "--depth", "1", EMOJI_REPO, LOCAL_CLONE_DIR], check=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)